src/vmware/azext_vmware/commands.py

```diff
@@ -96,6 +97,12 @@ def load_command_table(self, _):
         g.custom_command('update', 'addon_srm_update')
         g.custom_command('delete', 'addon_srm_delete')
 
+    with self.command_group('vmware addon arc', vmware_sdk, client_factory=cf_vmware) as g:
+        g.custom_command('create', 'addon_arc_create')
+        g.custom_show_command('show', 'addon_arc_show')
+        g.custom_command('update', 'addon_arc_update')
+        g.custom_command('delete', 'addon_arc_delete')

```

Comment:
Do we need to consider adding `confirmation=True` for the delete command?

Reply:
`az vmware addon arc delete` already has a delete confirmation